Tests/SwiftDocCTests/Semantics/MetadataAvailabilityTests.swift

```diff
@@ -90,17 +87,12 @@ class MetadataAvailabilityTests: XCTestCase {
             try assertValidAvailability(source: "@Available(\"My Package\", \(args))")
         }
 
-        // also test for giving no platform
-        for args in validArguments {
-            try assertValidAvailability(source: "@Available(\(args))")
-        }
```

Comment:
Removed this code since it was never called because `validArguments` was always empty before. However now we have `deprecated` as a value within the array